src/parameterized_problem.cpp

```diff
@@ -179,6 +179,11 @@ void init_disp(const Vector &x, Vector &u)
    u(u.Size()-1) = -0.2*x(0)*rdisp_f;
 }
 
+void init_disp_lcantilever(const Vector &x, Vector &u)
+{
+   u = 0.0;
+   u(u.Size()-1) = -0.2*(x(u.Size()-1) - 5.0)*rdisp_f;
```

Comment:
This comment doesn't have to be in this PR, just for a suggestion: I guess in the future we could add more parameters for `LineElastDispCantilever` that control `-0.2` and `-5.0` values here?

Reply:
Yes! The -5 value should not be hardcoded but correspond to w + h, where w and h are the width and length of the rod component respectively